src/pages/MenuPage.ts

```diff
@@ -5,24 +5,44 @@ export class MenuPage extends AbstractPage {
   readonly page: Page;
   readonly url: string;
   readonly coffeeItemLocators: Locator;
+  readonly addToCartConfirmButton: Locator;
+  readonly addToCartRejectButton: Locator;
 
   constructor(page: Page) {
     super(page);
     this.page = page;
     this.url = ''; 
     this.coffeeItemLocators = page.locator(`li:has(h4)`);
+    this.addToCartConfirmButton = page.getByRole('button', { name: 'Yes'});
+    this.addToCartRejectButton = page.getByRole('button', { name: 'No'});
   }
 
   async getPrice(name: string): Promise<number> {
     test.step(`Gets the price of ${name} cup`, async() => {});
     const headerString = await this.coffeeItemLocators.locator(`h4:has-text("${name} $")`).textContent() || '';
     return +headerString.split(' $')[1];
   }
-
+  
   async addToCart(name: string): Promise<void> {
```

Comment:
If you make it specific in the step name, maybe it is better to make it specific in the method name as well.

`addToCartByLeftClick`

Reply:
decided to call it 'addToCartByClick' (widely used method, so it is better to have shorter name) while adding to cart via confirmation dialog will not be used so often and includes 2 sequential actions: invokeAddToCartConfirmation -> confirmAddToCart